src/components/Global/Banner/MaintenanceBanner.tsx

```diff
@@ -0,0 +1,16 @@
+import { GenericBanner } from './GenericBanner'
+import config from '@/config/routesUnderMaintenance'
+
+interface MaintenanceBannerProps {
+    pathname: string
+}
+
+export function MaintenanceBanner({ pathname }: MaintenanceBannerProps) {
+    const isUnderMaintenance = config.routes.some((route) => pathname.startsWith(route))
+
+    if (!isUnderMaintenance) {
+        return null
+    }
+
+    return <GenericBanner message="Under maintenance" icon="⚠️" />
```

Comment:
thought: we should use our icons, but I believe that is changing from current version to pw

Reply:
good point!